modules/nw-multus-tap-object.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: REFERENCE
+[id="nw-multus-tap-object_{context}"]
+= Configuration for a TAP additional network
+
+The following object describes the configuration parameters for the TAP CNI
+plugin:
+
+.TAP CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `tap`.
+
+|`mac`
+|`string`
+|Optional: Request the specified MAC address for the interface.
+
+|`mtu`
+|`integer`
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+
+|`selinuxcontext`
+|`string`
+|Optional: The SELinux context to associate with the tap device.
+
+NOTE:
+The value `system_u:system_r:container_t:s0` is required for {product-title}.
+
+|`multiQueue`
+|`boolean`
+|Optional: Set to `true` to enable multi-queue.
+
+|`owner`
+|`integer`
+|Optional: The user owning the tap device
+
+|`group`
+|`integer`
+|Optional: The group owning the tap device.
+
+|`bridge`
+|`string`
+|Optional: Set the tap device as a port of an already existing bridge.
+|====
+
+[id="nw-multus-tap-config-example_{context}"]
+== Tap configuration example
+
+The following example configures an additional network named `mynet`:
+
+[source,json]
+----
+{
+ "name": "mynet",
+ "cniVersion": "0.3.1",
+ "type": "tap",
+ "mac": "00:11:22:33:44:55",
+ "mtu": 1500,
+ "selinuxcontext": "system_u:system_r:container_t:s0",
+ "multiQueue": true,
+ "owner": 0,
+ "group": 0
+ "bridge": "br1"
+}
+----
+
+[NOTE]
+====
+To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using machine config.
```

Comment:
container_use_device SELinux boolean on the host by using a machine config object. See https://docs.openshift.com/container-platform/4.13/nodes/nodes/nodes-nodes-managing.html#nodes-nodes-working-setting-booleans_nodes-nodes-managing.

Reply:
HI @mlguerrero12 thanks due to build restrictions we cannot add xref directly in a module. I have created another section Additional resources and added there, hopefully OK now for LGTM and get pushed into docs.

Reply:
by using the Machine Config Operator (MCO). I've seen this in other parts of the doc

Reply:
All done yes I was not sure but in section it refers to it is Machine Config Operator (MCO)